server/sonar-web/src/main/js/apps/marketplace/components/EditionBoxBadge.tsx

```diff
@@ -28,22 +28,34 @@ interface Props {
 }
 
 export default function EditionBoxBadge({ editionKey, status }: Props) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The function EditionBoxBadge has a cyclomatic complexity of 11 which is higher than the threshold of 10 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)